src/agent/src/main.c

```diff
@@ -1030,8 +1030,6 @@ bool GetConnectionInfoFromIdentityService(ADUC_ConnectionInfo* info)
     }
     memset(info, 0, sizeof(*info));
 
-    Log_Info("Requesting connection string from the Edge Identity Service");
-
```

Comment:
Note: Feel free to move **GetConnectionInfoFrom*** into **communication_abtraction** or [**create new**] communication_util/manager

Reply:
That's what I'm thinking of doing. Separate bug though